docs/build/getting-started/quickstarts/testing.md

```diff
@@ -0,0 +1,102 @@
+
+access(all) let calculator = Calculator()
+
+access(all) fun testAdd() {
+  Test.assertEqual(calculator.add(a: 2, b: 3), 5)
+}
+
+access(all) fun testSubtract() {
+  Test.assertEqual(calculator.subtract(a: 5, b: 3), 2)
+}
+
+```
+
+This code
+- imports the `Calculator` contract from the `calculator.cdc` file (according to `flow.json`); 
+- creates an `calculator` instance of the smart-contract;
+- defines two test functions: `testAdd()` and `testSubtract()`;
+- calls `add()` and `subtract()` methods with different input values  respectively.
+
+## Running the test cases
+
+To run the test cases, use the following command in the terminal:
+
+```bash
+flow test ./calculator_test.cdc
+```
+
+This command uses the Flow CLI to run the test cases and display the output. You should see the following output:
+
+```
+Test results: "./calculator_test.cdc"
+- PASS: testAdd
+- PASS: testSubtract
+```
+
+This output indicates that both test cases ran successfully, and the smart contract is functioning as expected.
+
+## Advanced Testing Techniques
+
+The Cadence testing framework provides various features and techniques for writing comprehensive test cases. Some of these include:
+
+- [**Code Coverage**](https://github.com/m-Peter/flow-code-coverage): You can use the `-cover` flag with the `flow test` command to view code coverage results when running your tests. This allows you to identify areas of your code that are not adequately covered by your test inputs;
+- **Test Fixtures**: Test fixtures are reusable components that help you set up the initial state for your test cases. You can create test fixtures in Cadence by defining resource types and using them in your test functions;
+- [**Assertions**](../../../cadence/testing-framework.mdx#assertions): The testing framework provides built-in assertion functions, such as `assertEqual`, `assertTrue`, and `assertFalse`, to help you verify the expected behavior of your smart contracts;
+- **Test Suites**: You can organize your test cases into test suites to improve the readability and maintainability of your test code. Test suites allow you to group related test cases and set up common test fixtures for all the tests in the suite.
+
+By leveraging these advanced testing techniques, you can write more robust and reliable smart contracts in Cadence.In this example, we set up a basic testing environment, wrote a simple smart contract in Cadence, and created a test case to verify its functionality. We then used the Flow CLI to run the test case and confirm that the smart contract is working correctly. This is a basic example, and there are many more advanced features and techniques you can explore when working with the Cadence testing framework. For more in-depth tutorials and documentation, refer to the official Flow Developer Portal and the Flow CLI documentation.
+
+## References
+
+- https://github.com/onflow/developer-grants/issues/216#issuecomment-1750673975
```

Comment:
```suggestion
- [Reference documentation for Cadence testing](../../../cadence/testing-framework.mdx)
```

I think sticking with the authoritative docs source is cleaner than linking to grant discussions